docs/admin/deploy/scale.mdx

```diff
@@ -336,25 +336,25 @@ Basically anything not related to code-intel.
 | :---------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `Overview`  | The Postgres instance will use memory by bringing OS pages into resident memory where it will control its own allocations                                                               |
 | `Factors`   | Size of all repositories                                                                                                                                                                |
 |             | Number of insight queries                                                                                                                                                               |
 | `Guideline` | Starts at default as the value grows depending on the number of active users and activity                                                                                               |
-|             | The database must be configured properly following our [Postgres configuration guide](https://docs.sourcegraph.com/admin/config/postgres-conf)to use the assigned resources efficiently |
+|             | The database must be configured properly following our [Postgres configuration guide](https://sourcegraph.com/docs/admin/config/postgres-conf)to use the assigned resources efficiently |
 | `Type`      | Persistent Volumes for Kubernetes                                                                                                                                                       |
 |             | Persistent SSD for Docker Compose                                                                                                                                                       |
 
-> WARNING: The database must be configured properly to consume resources effectively and efficiently for better performance and to avoid regular usage from overwhelming built-in utilities (like autovacuum for example). For example, a Postgres database running out of memory indicates that it is currently misconfigured and the amount of memory each worker can utilize should be reduced. See our [Postgres database configuration guide](https://docs.sourcegraph.com/admin/config/postgres-conf) for more information.
+> WARNING: The database must be configured properly to consume resources effectively and efficiently for better performance and to avoid regular usage from overwhelming built-in utilities (like autovacuum for example). For example, a Postgres database running out of memory indicates that it is currently misconfigured and the amount of memory each worker can utilize should be reduced. See our [Postgres database configuration guide](https://sourcegraph.com/docs/admin/config/postgres-conf) for more information.
```

Comment:
Please add the warning callout.:)

Reply:
I updated all of the `notes` and `warnings` in this doc with the new callout elements :)